### PR TITLE
Add default timezone for warehouse operations

### LINKED ISSUE #332: Warehouse Management - add timezone to config section
built off #29

### PR DESCRIPTION
**Pull request by rymurr**
closes #332

### CODE REVIEW

File: app/ui/pages/10_Settings.py [hunk 1]
```diff
@@ -82,6 +86,14 @@ def task_listing(
             key="serverless_credit_cost",
         )
         storage = st.text_input("Storage Cost (/tb)", value=tbcost, key="storage_cost")
+        timezone = st.selectbox(
+            "Default Timezone",
+            index=next(
+                i for i, v in enumerate(pytz.all_timezones) if v == timezone_start
```
Comment (joshelser) [inline app/ui/pages/10_Settings.py:92]: nit: Was glancing at `all_timezones` and see that it has over 500 entries in it. Would `common_timezones` be abrasive?

**Review by joshelser — APPROVED** [review]
I'm assuming that the timezone names that `pytz` returns are usable by Snowflake timezone types/functions, or maybe you're planning to do the core algorithm solely in Python and it doesn't matter if Snowflake's timezones align?

**Comment by rymurr** [comment]
> I'm assuming that the timezone names that `pytz` returns are usable by Snowflake timezone types/functions, or maybe you're planning to do the core algorithm solely in Python and it doesn't matter if Snowflake's timezones align?

Both use the standard notation fo rtimezones

File: app/ui/pages/10_Settings.py (hunk 1 above)
Comment (rymurr) [reply app/ui/pages/10_Settings.py:92]: fixed to use common_timezones. Makes a lot more sense

**Review by joshelser — APPROVED** [review]

**Pull request merged**